src/fu-tool.c

```diff
@@ -2133,6 +2134,9 @@ main (int argc, char *argv[])
 		{ "allow-older", '\0', 0, G_OPTION_ARG_NONE, &allow_older,
 			/* TRANSLATORS: command line option */
 			_("Allow downgrading firmware versions"), NULL },
+		{ "simulate", '\0', 0, G_OPTION_ARG_NONE, &simulate,
+			/* TRANSLATORS: command line option */
+			_("Simulate installation of the new firmware"), NULL },
```

Comment:
need to update bash completion (also for fwupdmgr if that one stays)

Reply:
Fixed.